Gruntfile.js

```diff
@@ -0,0 +1,128 @@
+module.exports = function (grunt) {
+    // Project configuration.
+    grunt.initConfig({
+        modx: grunt.file.readJSON('_build/config.json'),
+        sshconfig: grunt.file.readJSON('/Users/jako/Documents/MODx/partout.json'),
```

Comment:
hardcoded?

Reply:
I have changed that part. It does not upload the files with SFTP anymore. The grunt process does not have any local development code anymore.